pkg/chartverifier/checks/checks.go

```diff
@@ -337,3 +361,40 @@ func parseImageReference(image string) pyxis.ImageReference {
 	return imageRef
 
 }
+
+func getOCPRange(kubeVersionRange string) string {
+
+	semverCompare := sprig.GenericFuncMap()["semverCompare"].(func(string, string) (bool, error))
+	minOCPVersion := ""
+	maxOCPVersion := ""
+	for kubeVersion, OCPVersion := range tool.GetKubeOpenShiftVersionMap() {
+		match, err := semverCompare(kubeVersionRange, kubeVersion)
+		if err != nil {
+			return fmt.Sprintf("%s : %s", KuberVersionProcessingError, err)
+		}
+		if match {
+			testOCPVersion := fmt.Sprintf("v%s", OCPVersion)
+			if minOCPVersion == "" || semver.Compare(testOCPVersion, fmt.Sprintf("v%s", minOCPVersion)) < 0 {
+				minOCPVersion = OCPVersion
+			}
+			if maxOCPVersion == "" || semver.Compare(testOCPVersion, fmt.Sprintf("v%s", maxOCPVersion)) > 0 {
+				maxOCPVersion = OCPVersion
+			}
+		}
+	}
+	if minOCPVersion != "" {
+		match, _ := semverCompare(kubeVersionRange, "1.999")
```

Comment:
Confused in this condition check, why are we comparing to 1.999 here?

Reply:
Got it. Thanks for explaining! I think a comment as a note will be helpful.

Reply:
Comment now added.